### PR TITLE
Minor bugfixes

### PR DESCRIPTION
**Pull request by ClemensKohl**
Nothing major, I just corrected some minor bugs.

### CODE REVIEW

File: vignettes/CAclust.Rmd [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- name: Clemens Kohl
+  affiliation: Max Planck Institute for Molecular Genetics, Berlin, Germany
+  email: kohl@molgen.mpg.de
+- name: Yan Zhao
+  affiliation: Max Planck Institute for Molecular Genetics, Berlin, Germany
+  email: zhao@molgen.mpg.de
+- name: Martin Vingron
+  affiliation: Max Planck Institute for Molecular Genetics, Berlin, Germany
+  email: vingron@molgen.mpg.de
+package: APL
```
Comment (yanzhaobiomath) [inline vignettes/CAclust.Rmd:13]: APL --> CAniNET

**Review by yanzhaobiomath — COMMENTED** [review]
A small typo found

**Comment by ClemensKohl** [comment]
Thanks for spotting it, I changed it to CAbiNet. We should probably eventually rename the package too.

**Review by yanzhaobiomath — APPROVED** [review]
:+1:

**Pull request merged**